services/121-service/package.json

```diff
@@ -32,10 +32,10 @@
   "private": true,
   "dependencies": {
     "@nestjs/axios": "^3.0.2",
-    "@nestjs/passport": "^10.0.3",
     "@nestjs/bull": "^10.1.1",
     "@nestjs/common": "^10.3.8",
     "@nestjs/core": "^10.3.8",
+    "@nestjs/passport": "^10.0.3",
```

Comment:
This thing weirdly jumps around a few times...

I wonder if everyone is using the same Node version?
Or that there are differences in `npm` running on macOS or Windows...
Or that Dependabot has an influence on it?

Reply:
Indeed, I don't get it either. I do think it's dependabot moving it around manually, rather than using npm install. I'm not too fussed about it though tbh :D